test/e2e/util.go

```diff
@@ -234,6 +234,26 @@ func SkipUnlessNodeCountIsAtLeast(minNodeCount int) {
 	}
 }
 
+func SkipIfMajorMinorVersionIs(c client.VersionInterface, major, minor string) {
+	info, err := c.ServerVersion()
+	if err != nil {
+		Skipf("Cowardly skipping due to error determining version: %v", err)
```

Comment:
This seems like a recipe for overlooked problems; I'd much rather fail than pass.